src/itemsTotal.js

```diff
@@ -0,0 +1,10 @@
+import { getData } from './api.js';
+
+const itemTotal = async () => {
+  const total = await getData();
+  const items = document.querySelector('.items');
+  items.textContent = `Total Disney Characters: ${total.length} Characters`;
+};
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- Hey, as it's highly not recommended by microverse to disable the linter, please, let's try to write codes that conform to the linter as this would help us in writing clean and maintainable code. Please, kindly fix this linter by adding the default keyword and update the index.js where it's been imported to 
`import itemTotal from './itemsTotal.js';`

Reply:
Done